addons/html_builder/static/src/builder/builder_components/utils.js

```diff
@@ -440,12 +440,14 @@ export function useInputBuilderComponent() {
     const getAction = comp.env.editor.shared.builderActions.getAction;
     const state = useDomState(getState);
     const applyValue = comp.env.editor.shared.history.makePreviewableOperation((value) => {
+        const action = !value || !parseInt(value) ? "clean" : "apply";
```

Comment:
I don't see why an input component should use a clean.
There should always be a value provided to the function `applyValue`.